tox.ini

```diff
@@ -12,6 +12,13 @@ commands = py.test
 deps =
     -r{toxinidir}/requirements.txt
 
+[testenv:coverage]
+basepython = python3
```

Comment:
Change the python version to 3.7, please.

Reply:
Done